fix warning in test_footprinter.cc

Fix following warning:
```
/vt/lib/checkpoint/tests/unit/test_footprinter.cc:117:9: warning: declaration shadows a local variable [-Wshadow]
    int i = 3;
        ^
/vt/lib/checkpoint/tests/unit/test_footprinter.cc:80:7: note: previous declaration is here
  int i;
      ^
```

--- a/tests/unit/test_footprinter.cc
+++ b/tests/unit/test_footprinter.cc
@@ -114,8 +114,8 @@ TEST_F(TestFootprinter, test_fundamental_types) {
   }
 
   {
-    int i = 3;
-    void* ptr = static_cast<void*>(&i);
+    int j = 3;
+    void* ptr = static_cast<void*>(&j);
     EXPECT_EQ(
       checkpoint::getMemoryFootprint(ptr),
       sizeof(ptr)

